tools/otci/otci/otci.py

```diff
@@ -99,6 +99,9 @@ def execute_command(self, cmd: str, timeout: float = 10, silent: bool = False) -
         if not silent:
             self.__logger and self.__logger.info('> %s', cmd)
 
+        if "dns" in cmd:
```

Comment:
Can we use a different timeout in methods below?
- dns_browse
- dns_resolve
- dns_resolve_service

It seems a little hardcoding to treat `dns` commands differently in `execute_command`?

Reply:
Sure will do that way.